test/components/generators/chat/test_hugging_face_local.py

```diff
@@ -78,6 +79,15 @@ def custom_tool_parser(text: str) -> Optional[List[ToolCall]]:
 
 
 class TestHuggingFaceLocalChatGenerator:
+    @pytest.fixture(autouse=True)
+    def mock_model_info(self, monkeypatch):
+        """Mock the model_info function to prevent real HTTP requests."""
+
+        def mock_model_info(*args, **kwargs):
+            return {"pipeline_tag": "text2text-generation"}
+
+        monkeypatch.setattr("huggingface_hub.HfApi.model_info", mock_model_info)
```

Comment:
1. this seems like a duplicate `model_info_mock`
2. I would avoid `autouse=True` and explicitly use `model_info_mock` where needed